items/PERFECT_ONYX_GEM.json

```diff
@@ -14,7 +14,7 @@
     "§6§lLEGENDARY"
   ],
   "internalname": "PERFECT_ONYX_GEM",
-  "crafttext": "",
+  "crafttext": "Requries: Gemstone X",
```

Comment:
Doesn't this also require HotM 5 or something?

Reply:
> Doesn't this also require HotM 5 or something?

Wiki doesnt mention anything about a hotm req
also ingame: 
![image](https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/assets/69345714/8b5b8e66-8e22-4543-a96c-ef638fa529ba)

Reply:
![image](https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/assets/32849887/347670f7-a1ef-49e7-b59d-d4f646eedeb2)

Also Tungsten Plate, Umber Plate, and Glacite Amalgamation.

Reply:
https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/assets/32849887/8583c5a1-60d8-479b-88f2-64b0b77a3061

![image](https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/assets/32849887/9134d179-d26a-4127-ad64-46c1cfe5f781)

![image](https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/assets/32849887/97f2eb4f-0a1e-489f-a6b5-7f2833efc4e3) ![image](https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/assets/32849887/2605580d-9c87-4ab8-8849-db732915e7cc) ![image](https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/assets/32849887/76f3aa3a-bada-4441-bdea-69b9429e4d78)

Reply:
I dislike them really hard.

Reply:
Will fix it tmr probably, im too tired for this.

Reply:
https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/commit/ac1a1a146c6a1cbcd5208e49384a39777a3bd11e